clang/lib/Basic/Targets/NVPTX.h

```diff
@@ -180,6 +172,18 @@ class LLVM_LIBRARY_VISIBILITY NVPTXTargetInfo : public TargetInfo {
     TargetInfo::adjust(Opts);
     // FIXME: Needed for compiling SYCL to PTX.
     TLSSupported = TLSSupported || Opts.SYCLIsDevice;
+
+    if (Opts.SYCLIsDevice) {
```

Comment:
Should this work during host compilation?

Reply:
This should only affect device-side.